framework/source/class/qx/ui/table/columnmenu/MenuItem.js

```diff
@@ -29,12 +29,12 @@ qx.Class.define("qx.ui.table.columnmenu.MenuItem",
     /**
      * Whether the table column associated with this menu item is visible.
      */
-    visible :
+    columnVisible :
     {
       check : "Boolean",
       init  : true,
-      apply : "_applyVisible",
-      event : "changeVisible"
+      apply : "_applyColumnVisible",
+      event : "changeColumnVisible"
     }
```

Comment:
Fixing this _without changing the API_ is difficult:
Either `qx.ui.core.Widget#isVisible()` must be renamed (API change),
...or `qx.ui.table.columnmenu.MenuItem#visible` must be renamed (API change)

What might be a solution is the following:
Changing the Interface in a way that keeps any implementing class happy:
```
qx.Interface.define("qx.ui.table.IColumnMenuItem",
{
  events :
  {
    /**
     * Dispatched when a column changes visibility state. The event data is a
     * boolean indicating whether the table column associated with this menu
     * item is now visible.
     */
    changeVisible : "qx.event.type.Data"
  },

  members :
  {
    // Get/Set whether the table column associated with this menu item is visible
    setVisible : function (value) {},
    getVisible : function () {}
  }
});
```
With this, any implementing class that used a property named `visible`, still fulfills the interface's requirements because `setVisible` method, `getVisible` method and `changeVisible` event are auto-created[1]!

What do you think?

[1] Such an application class will raise the warning "Conflicting property method what.ever.Clazz.isVisible with qx.ui.core.Widget", but that's (a) correct and (b) fixable by the application-developer